apps/src/craft/craft.js

```diff
@@ -281,6 +281,10 @@ Craft.init = function (config) {
       if (!config.level.showPopupOnLoad) {
         Craft.initializeAppLevel(config.level);
       }
+
+      // Adjust visualizationColumn width.
+      var visualizationColumn = document.getElementById('visualizationColumn');
+      visualizationColumn.style.width = this.nativeVizWidth + 'px';
```

Comment:
Good catch. You're right, this messed with resizing in the non-sharing view. I updated this to only apply to sharing.